Pokemon Unity/Assembly-CSharp-Editor.csproj

```diff
@@ -45,76 +45,76 @@
     <Reference Include="System.Runtime.Serialization" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="UnityEngine">
-      <HintPath>C:/Program Files/Unity/Editor/Data/Managed\UnityEngine.dll</HintPath>
+      <HintPath>F:/Program Files/Unity/Editor/Data/Managed\UnityEngine.dll</HintPath>
     </Reference>
```

Comment:
Lol, you're committing your local changes back to global repository. Is it possible for you to `cherry pick` (revert) these out?